Cilsil/Cil/Parsers/LeaveParser.cs

```diff
@@ -20,9 +20,20 @@ internal class LeaveParser : InstructionParser
                 // Leave for catch block: target instruction immediately after catch block.
                 case Code.Leave:
                 case Code.Leave_S:
+                case Code.Throw:
```

Comment:
Why `Throw` is handled in `LeaveParser`? Didn't we have a `ThrowParser` already?